keyboards/bluebell/swoop/keymaps/default/keymap.c

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2022 James White <jamesmnw@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _QWERTY
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+/*
+ * QWERTY
+ * ,---------------------------------.     ,----------------------------------.
+ * |  Q  |   W  |   E  |   R  |   T  |     |   Y  |   U  |   I  |   O  |   P  |
+ * |-----+------+------+------+------|     |------+------+------+------+------|
+ * |  A  |   S  |   D  |   F  |   G  |     |   H  |   J  |   K  |   L  |   ;  |
+ * |-----+------+------+------+------|     ,------+------+------+------+------|
+ * |  Z  |   X  |   C  |   V  |   B  |     |   N  |   M  |   ,  |   .  |   /  |
+ * `------------+------+------+------|     |------+------+------+-------------'
+ *              | LCTL | BSpc | Esc  |     | Ent  | Spc  | LAlt |
+ *              `--------------------'     `--------------------'
+ */
+
+[_QWERTY] = LAYOUT(
```

Comment:
```suggestion
[_QWERTY] = LAYOUT_split_3x5_3(
```

And same with kyek layout.